src/structures/BotClient.js

```diff
@@ -63,7 +63,7 @@ module.exports = class BotClient extends Client {
       : undefined;
 
     // Music Player
-    if (this.config.MUSIC.ENABLED) this.musicManager = lavaclient(this);
+    if (this.config.MUSIC.ENABLED) this.manager = new Manager(this);
```

Comment:
```suggestion
    if (this.config.MUSIC.ENABLED) this.musicManager = new Manager(this);
```
I think this should be still named `musicManager` as it's informative, rather than the plain `manager` which tells you nothing. Otherwise great PR